.github/workflows/ci.yml

```diff
@@ -116,6 +119,29 @@ jobs:
     - name: Run migrations
       run: ./tobira db migrate --config util/dev-config/config.toml
 
+    # UI tests
+    - name: Start docker containers
+      working-directory: util/containers
+      run: |
+        docker-compose -f docker-compose.yml up -d \
+          tobira-auth-proxy \
+          tobira-login-handler \
+          tobira-meilisearch
+
+    - name: Rebuild search index
+      working-directory: backend
+      run: echo yes | cargo run -- search-index update
+
+    - name: Run playwright tests
+      working-directory: frontend
+      run: npx playwright test
+    - name: Upload test results
+      if: always()
```

Comment:
Does it make sense to always upload the results? Feels like it only makes sense if the test failed? Or what is this good for?

Reply:
Ah no, your explanation makes sense. And it's a really tiny file anyway, so let's keep it like this. But maaaybe add a very brief comment there to explain it? Not super important tho.